backstop/tests/non-ideal-state.html

```diff
@@ -1845,7 +1845,7 @@
       class="iui-non-ideal-state-scrolling"
     >
       <div class="iui-non-ideal-state">
-        <!-- Use error.svg from Bentley/Illustrations package. -->
+        <!-- Use status-error.svg from @itwin/itwinui-illustrations package. -->
```

Comment:
There is no such svg in illustrations...

Reply:
Must have got accidentally changed with a Find & Replace.  Reverted.